omnibus/files/private-chef-cookbooks/private-chef/templates/default/nginx/nginx_chef_api_lb.conf.erb

```diff
@@ -44,6 +44,13 @@
       default_type text/plain;
       alias /opt/opscode/version-manifest.txt;
     }
+    <% if node['private_chef']['nginx']['enable_lets_encrypt'] -%>
```

Comment:
@alexbacchin My understanding is that the certbot util uses port 80, so this `location` block (and the config flag) could actually go away again, right?